diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 891302ed..32b2d885 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -72,12 +72,12 @@ if (_GLFW_X11 OR _GLFW_WAYLAND)
     endif()
 endif()
 
-if (APPLE)
-    # For some reason CMake didn't know about .m until version 3.16
-    set_source_files_properties(cocoa_init.m cocoa_joystick.m cocoa_monitor.m
-                                cocoa_window.m nsgl_context.m PROPERTIES
-                                LANGUAGE C)
-endif()
+#if (APPLE)
+#    # For some reason CMake didn't know about .m until version 3.16
+#    set_source_files_properties(cocoa_init.m cocoa_joystick.m cocoa_monitor.m
+#                                cocoa_window.m nsgl_context.m PROPERTIES
+#                                LANGUAGE C)
+#endif()
 
 add_library(glfw ${glfw_SOURCES} ${glfw_HEADERS})
 set_target_properties(glfw PROPERTIES
